msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/MainActivity.java

```diff
@@ -20,6 +20,7 @@
 //  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
+
```

Comment:
This here is to trigger the `test-automation` label.

Reply:
Feel free to revert this before you commit. I believe you can use `git commit --allow-empty -m "Empty commit"` to trigger builds, tag-events, etc in the future.

Cheers!

Reply:
Thanks for this tip @iambmelt.